start.sh

```diff
@@ -116,11 +116,12 @@ EOI
     cat << EOI
 
 ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ IPFS settings                                                                               ┃
+┃ Do you want to change the IPFS settings [y/N]:                                              ┃
+┃ [When using Testnet, you must change the IPFS Provider information]                         ┃
```

Comment:
Noting a connection to the update on the SAT side of this: https://github.com/ProjectLibertyLabs/social-app-template/pull/146/files#r1716891837